dist/js/main.js

```diff
@@ -0,0 +1,15 @@
+const burgerBtn = document.querySelector('.burger');
+const about = document.querySelector('.burger-content');
+about.style.top = '-2500px';
+
+burgerBtn.addEventListener('click', () => {
+  if (about.style.top === '-2500px') {
```

Comment:
we can't rely on style attributes. Another script could change it to 3px and BOM